add validation webhook example

barisozcetin opened the pull request:
## Description

Added validation webhook example 

[ ] Descriptive folder name
[ ] No unnecessary files and code
[ ] README file
[ ] Added to top level README

TomPridham reviewed (approved):
should you fill out the readme or is brandon going to do that?

barisozcetin commented:
> should you fill out the readme or is brandon going to do that?

added some info in the readme. I'll check with Brandon if it's enough. also updated main readme
